Index.html

```diff
@@ -20,21 +20,159 @@
+    </section>
+    <section class="section">
+        <span><img src="./images/Snapshoot3.svg"></span>
+        <h2>Tonic</h2>
+        <span class="sublinks">
+            <span>CANOPY</span>
+            <ul>
+                <li><a href="#">Bank End Dev</a></li> 
+                <li><a href="#">2015</a></li> 
+                          
+             </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+      <div><a href="" class="seeproject">See Project</a></div>
+    </section>
+    <section class="section">
+        <span><img src="./images/Snapshoot4.svg"></span>
+        <h2>Multi-Post Stories</h2>
+        <span class="sublinks">
+            <span>CANOPY</span>
+            <ul>
+                <li><a href="#">Bank End Dev</a></li> 
+                <li><a href="#">2015</a></li> 
+                          
+             </ul>
+        </span>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+      <div><a href="" class="seeproject">See Project</a></div>
+    </section>
+    <section class="section">
+        <span>&nbsp;</span>
+        <span>&nbsp;</span>
+        <h2>About Myself</h2>
+        
+        <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <span class="sublinksround">
+            
+            <ul>
+                <li><a href="#">HTML</a></li> 
+                <li><a href="#">CSS</a></li> 
+                <li><a href="#">JavaScript</a></li>          
+             </ul>
+        </span>
+     
+    </section>
+    
+    <section class="section">
+        <span class="connect"><a href="#" > LET’S CONNECT</a></span>
+        <ul class="socials">
            <li><a href="#"><img src="./images/twitter.png"></a></li>
            <li><a href="#"><img src="./images/linkedin.svg"></li>
            <li><a href="#"><img src="./images/m.png"></a></li>
            <li><a href="#"><img src="./images/git.png"></a></li>
            <li><a href="#"><img src="./images/peace.png"></a></li>
         </ul>
-    </footer>
+        <div><a href="" class="seeproject">Get my resume</a></div>
+    </section>
+    <section class="section">   
+        <h3>Languages</h3>
```

Comment:
- Kindly add the arrow near the languages as it is shown in the picture below it will make your design stick more to the figma design :+1: 

![image](https://user-images.githubusercontent.com/95297251/227676107-659e9ea2-5ce7-4ebf-b306-c8489f994865.png)